package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "omt-odt-language",
-    "version": "2.3.0",
+    "version": "2.4.0",
```

Comment:
The package lock should also be updated, because it has the version tag in it as well.